inc/az_ulib_ipc_api.h

```diff
@@ -150,6 +172,41 @@ AZ_NODISCARD az_result az_ulib_ipc_publish(
     const az_ulib_interface_descriptor* const interface_descriptor,
     az_ulib_ipc_interface_handle* interface_handle);
 
+/**
+ * @brief   Make an interface default in the device.
+ *
+ * This API sets an interface implementation as default in the device, so if 2 or more packages
+ * implement the same interface, IPC will be able to define each one shall be returned, if the
+ * caller didn't specify the desired package.
+ *
+ * If another package was already defined as default for the interface, call this function will
+ * change the default, from the previous package, to the provided one.
```

Comment:
nit: "If another package was already defined as default for the given interface, calling this API will change the default from the previous package to the provided one."